sdk/core/Azure.Core/samples/ProtocolMethods.md

```diff
@@ -92,65 +92,55 @@ Protocol methods need a JSON object of the shape required by the schema of the s
 See the specific service documentation for details, but as an example:
 
 ```csharp
-	var data = new {
-                name = "Buddy",
-                id = 2,
-                color = "Brown"
-        };
-        /*
-        {
-            "name": "Buddy",
-            "id": 2,
-            "color": "Brown"
-        }
-        */
-        client.SetDog(RequestContent.Create(data));
+var data = new {
```

Comment:
Let's use a code snippet if possible.

Reply:
Tracked in https://github.com/Azure/azure-sdk-for-net/issues/28315